src/@types/js-xdr/index.d.ts

```diff
@@ -4,12 +4,12 @@
 // Timur Ramazanov <charlie.vmk@gmail.com>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.4.1
-declare module 'js-xdr' {
-  import * as Long from 'long'
+declare module "js-xdr" {
+  import * as Long from "long";
 
   export class Array {
-    _childType: any
-    _length: number
+    public _childType: any;
```

Comment:
Can we set more strict type here? It seems that `_childType` should implement `read`, `write` and `isValid` methods

Reply:
@charlie-wasp  👍 will do!